migtests/tests/oracle/assessment-report-test/expectedAssessmentReport.json

```diff
@@ -180,7 +180,8 @@
 			"FeatureName": "Compound Triggers",
 			"ObjectNames": [
 				"trg_simple_insert"
-			]
+			],
+			"DDLs": null
```

Comment:
You will probably need to change these json outputs as well.

Reply:
Yes will do